packages/mask/src/plugins/Collectible/SNSAdaptor/OrderRow.tsx

```diff
@@ -139,20 +145,25 @@ export function OrderRow({ order, isDifferenceToken }: IRowProps) {
                                 order.payment_token_contract?.decimals,
                                 order.quantity,
                             )} ${
-                                provider === CollectibleProvider.OPENSEA
+                                provider === NonFungibleAssetProvider.OPENSEA
                                     ? order.payment_token_contract?.symbol ?? ''
                                     : 'ETH'
                             }`}
                         </Typography>
                     </TableCell>
-                    {provider === CollectibleProvider.OPENSEA ? (
+                    {provider === NonFungibleAssetProvider.OPENSEA ? (
                         <TableCell>
                             <Typography className={classes.content}>
                                 {order.expiration_time &&
                                     !isZero(order.expiration_time) &&
-                                    formatDistanceToNow(new Date(order.expiration_time * 1000), {
-                                        addSuffix: true,
-                                    })}
+                                    formatDistanceToNow(
+                                        new Date(
+                                            new BigNumber(order.expiration_time ?? 0).multipliedBy(1000).toNumber(),
```

Comment:
`order.expiration_time && !isZero(order.expiration_time) && <EXPR>` is equivalent to `order.expiration_time && <EXPR>`

Reply:
```suggestion
                                            new BigNumber(order.expiration_time ?? 0).multipliedBy(1000).toNumber(),
                                            +order.expiration_time * 1000
```


No need to use `BigNumber`.

Also, better to multiply 1000 in data processing layer than in UI layer, like what you did here:

https://github.com/DimensionDev/Maskbook/blob/31b6821d0d7f3bd0a157c258c831148e2f217b2d/packages/web3-providers/src/NFTScan/index.ts#L26-L27